.github/workflows/openapi-docs-pr.yml

```diff
@@ -0,0 +1,16 @@
+name: Test OpenAPI doc is valid on Pull Requests
+
+on:
+  pull_request:
+    paths:
+      - "docs/**"
+jobs:
+  build:
+    name: Run Spectral
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: stoplightio/spectral-action@v0.8.0
+        with:
+          file_glob: 'docs/openapi.yml'
+          spectral_ruleset: 'docs/.spectral.yml'
```

Comment:
We could manage this by changing the working directory, so we're in the right location?

Reply:
oh sorry I misunderstood this, I thought you meant moving this to the schemas directory but just realised you meant changing the github workflow - I think I'm going to have to put this back in a separate directory because now the JsonSchema2Pojo plugin is trying to create a Java object from it